src/tools/TerminalStress/TerminalStress.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This is different from upstream, but I'm doing it to keep our package versions across the repo in sync